pkg/kubelet/dockershim/docker_container.go

```diff
@@ -154,6 +146,14 @@ func (ds *dockerService) CreateContainer(podSandboxID string, config *runtimeapi
 		applyContainerSecurityContext(lc, podSandboxID, createConfig.Config, hc)
 	}
 
+	// Apply cgroupsParent derived from the sandbox config.
+	if lc := sandboxConfig.GetLinux(); lc != nil {
```

Comment:
The previous if clause is `ContainerConfig.GetLinux()` but this part is` sandboxConfig.GetLinux()`, can we really combine them together? (i.e. those two need to be checked at same time?)